src/store/modules/product/actions.ts

```diff
@@ -83,6 +83,51 @@ const actions: ActionTree<ProductState, RootState> = {
       this.dispatch('product/fetchProducts', { productIds })
       this.dispatch('stock/addProducts', { productIds })
     }
+  },
+
+    /**
+   * Get Product Inventory
+   */
+  async getProductInventory({ commit, state }, payload) {
```

Comment:
This action is not returning product inventory, it's just returning the products information so we can improve it's name to something like fetchProducts

Reply:
Changed name of the function.